spec/factories/event_schedule.rb

```diff
@@ -2,6 +2,7 @@
 
 FactoryGirl.define do
   factory :event_schedule do
+    start_time DateTime.parse("#{Date.today.to_s} 00:00").utc
```

Comment:
You can also write it like that, more readable in my opinion:

`DateTime.now.utc.beginning_of_day`

But this is wrong, conference start_hour is 9, so you can not schedule it at 00.

Reply:
ok, fine